src/main/webapp/ts/app.component.ts

```diff
@@ -58,15 +58,20 @@ import {PayloadComponent} from './payload';
             segments[segments.length - 1].getPayload() : null'></payload>
         </div>
       </div>
-      <footer>
```

Comment:
Why are we removing the footer? Can you please share a screenshot of how it's effecting the UI?

Reply:
![screen shot 2018-05-11 at 8 11 20 am](https://user-images.githubusercontent.com/23624214/39896796-f849294a-54f2-11e8-86a9-573318dcc492.png)

I could not get the UI how I wanted it with the footer the way it was.